src/bridge/SequencerInbox.sol

```diff
@@ -38,10 +38,11 @@ import "../rollup/IRollupLogic.sol";
 import "./Messages.sol";
 import "../precompiles/ArbGasInfo.sol";
 import "../precompiles/ArbSys.sol";
+import "../libraries/I4844Readers.sol";
 
 import {L1MessageType_batchPostingReport} from "../libraries/MessageTypes.sol";
-import {GasRefundEnabled, IGasRefunder} from "../libraries/IGasRefunder.sol";
-import "../libraries/DelegateCallAware.sol";
```

Comment:
also this seems to be removed by accident

Reply:
[e3cc721](https://github.com/OffchainLabs/nitro-contracts/pull/111/commits/e3cc7214d4f734d06d3476798e641ae68bbf5ce8)